mappings/net/minecraft/client/gui/screen/realms/RealmsMainScreen.mapping

```diff
@@ -15,6 +16,7 @@ CLASS net/minecraft/unmapped/C_uncfwjlr net/minecraft/client/gui/screen/realms/R
 	FIELD f_jtfehbdt LOGO_HEIGHT I
 	FIELD f_kjcwkfau leaveButton Lnet/minecraft/unmapped/C_buwziidm;
 	FIELD f_kmjtkmrs LOADING Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_laokacvd unstable Z
```

Comment:
there are two fields named "unstable" in this class, which I don't love
mojang names this `snapshotToggle` and yarn `showingSnapshotRealms`

I also prefer the `snapshot` terminology over `unstable`